rest/route/degraded_mode.go

```diff
@@ -43,9 +45,14 @@ func (h *degradedModeHandler) Run(ctx context.Context) gimlet.Responder {
 	if err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving service flags"))
 	}
-	flags.CPUDegradedModeDisabled = false
-	if err = flags.Set(ctx); err != nil {
-		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "setting service flags"))
+	if flags.CPUDegradedModeDisabled {
+		flags.CPUDegradedModeDisabled = false
+		if err = flags.Set(ctx); err != nil {
+			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "setting service flags"))
+		}
+		grip.Info(message.Fields{
+			"message": "degraded mode has been triggered",
```

Comment:
nit: `CPU degraded mode has been triggered` for consistency.

Can we also include the route name here?